pepper-apis/src/main/resources/db-changes/schema/DDP-7417-add-matrix-modal-columns.xml

```diff
@@ -8,6 +8,7 @@
         <addColumn tableName="matrix_question">
             <column name="render_modal" type="boolean" defaultValueBoolean="false"/>
             <column name="modal_template_id" type="bigint" />
+            <column name="modal_title_template_id" type="bigint" />
```

Comment:
Once a Liquibase changeset has been pushed to `develop`, it can not be further modified- Liquibase will either fail the migration, or ignore the change completely. This column addition should be implemented as a new changeset.